python/ray/data/_internal/planner/map_batches.py

```diff
@@ -32,7 +34,14 @@ def fn(
 
         def validate_batch(batch: Block) -> None:
             if not isinstance(
-                batch, (list, pa.Table, np.ndarray, dict, pd.core.frame.DataFrame)
+                batch,
+                (
+                    list,
+                    pa.Table,
+                    np.ndarray,
```

Comment:
np.ndarray should not allowed for strict mode?

Reply:
See above (don't think we need to check here).